packages/flutter/lib/src/material/choice_chip.dart

```diff
@@ -158,7 +165,7 @@ class ChoiceChip extends StatelessWidget
       onSelected: onSelected,
       pressElevation: pressElevation,
       selected: selected,
-      showCheckmark: Theme.of(context).useMaterial3,
+      showCheckmark: showCheckmark ?? Theme.of(context).useMaterial3,
```

Comment:
```suggestion
      showCheckmark: showCheckmark ?? chipTheme.showCheckmark ?? Theme.of(context).useMaterial3,
```


Please add `chipTheme.showCheckmark`.

Since we're defaulting here instead of leaving it for `RawChip` to handle. We should have added the theme value before the default value.

Reply:
You're right. I'll add chipTheme.showCheckmark before the default Theme.of(context).useMaterial3.